spec/lib/whitehall_importer/integrity_checker_spec.rb

```diff
@@ -162,7 +162,7 @@
       )
 
       integrity_check = WhitehallImporter::IntegrityChecker.new(edition)
-      expect(integrity_check.problems).to include("organisations don't match")
+      expect(integrity_check.problems).to include(match("organisations don't match"))
     end
```

Comment:
No too sure of the answer, as we have lot of repetition across the tests - but it feels like at least once we should be testing the description message is exactly what we expect. Not sure if now is maybe the time to add a shared helper, what do you think?

Reply:
I've attempted to refactor this and reduce some repetition in a new commit https://github.com/alphagov/content-publisher/pull/1630/commits/484ba19c4b17adc7f7b9f5dc8450d477b9e76d2d.  Let me know what you think.